resources/puppetlabs/lein-ezbake/staging-templates/project_data.yaml.mustache

```diff
@@ -25,6 +25,8 @@ templates:
     target: ext/debian/{{{project}}}.default
   - source: ext/debian/ezbake.init.erb
     target: ext/debian/{{{project}}}.init
+  - source: ext/debian/ezbake.service.erb
+    target: ext/debian/{{{project}}}.service
```

Comment:
it's being installed by dh_installinit as part of the build, not install.sh (`dh_installinit -ppuppetserver` is called in the build).

https://wiki.debian.org/Teams/pkg-systemd/Packaging#systemd_unit_files_naming_and_installation